examples/CommonGateway/c-leds/readme.md

```diff
@@ -1 +1,60 @@
-# To Do as per [template](../../examples_readme_blueprint.md)
\ No newline at end of file
+# Example Project: C-LEDS for CommonGateway Development Kit
+
+## Goal:
+This example is the hello world example for running Kiso on the CommonGateway development kit.
+
+
+## Brief:
+The example will flash the blue LED. It will also show how to enable the logging module.
+
+### Components Required:
+
+#### Software Components
+Please refer to the getting started [quick start](http://kiso.rempler.de:1313/user-guide/quick_start.html) guide for Eclipse Kiso. 
+
+#### Hardware Components
+You will need CommonGateway board.If you want to flash, you need an external debugger,please refer to the vendors reference manual.
```

Comment:
Flashing today requires a J-Link debugger, to which vender? Bosch for the common gateway or the debugger's vendor?

Reply:
Done